tests/golos.referral_tests.cpp

```diff
@@ -47,8 +47,8 @@ class golos_referral_tester : public extended_tester {
         const string limit_persent     = amsg("max_percent > 100.00%");
     } err;
 
-    const asset min_breakout = asset(10000,  symbol(4,"GLS"));
-    const asset max_breakout = asset(100000, symbol(4,"GLS"));
+    const asset min_breakout = asset(10000,  symbol(3,"GLS"));
```

Comment:
For me it's simpler to use `token.make_asset` helper and forget about `symbol`. it's not needed to install token contract for that, just initialize helper